_pages/dinner.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Self-organised dinner
+snake: rainbow
+permalink: /dinner/
+---
+
+Ever found yourself in a new town, or at a new conference, feeling social but not knowing how to find people to hang out with?
+
+Care about making PyCon a welcoming place for newcomers?
+
+The self-organised dinner is just a designated time (just after the lightning talks) and place (just outside the Cockle Bay room) for people to self-organise into groups for dinner.
+
+This is how it will play out (once we've introduced the plan):
+1. People will mingle for a while, waiting for groups and group-leaders to emerge.
+2. Eventually a couple of people will suggest finding a restaurant (they are now the group leaders).
+3. A handy list of nearby restaurants that can fit groups will be consulted.
+4. A group will choose a restaurant and depart.
+5. All remaining people will continue to form groups and depart until no one is left.
+
+Lists of nearby restaurants which can fit groups will be shared around but feel free to disregard the list and choose your own. Saturday night can be busy in Sydney, but if you aim for an early dinner (6:30) you should be able to avoid the worst of it.
```

Comment:
```suggestion
Lists of nearby restaurants which can fit groups will be shared around but feel free to disregard the list and choose your own. Saturday night can be busy in Sydney, but if you aim for an early dinner (6:30pm) you should be able to avoid the worst of it.
```